### ISSUE TITLE
(WCAG 2.1) Lack of word 'error' or an error icon in front of the error message

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5484]
OBSERVED RESULTS:

- When an error occurs in any of the form fields, the error message is displayed in red text.
- There is no visual indication that the message is an error message, such as an error icon or the word 'error' present in front of the message.
- Users with color vision deficiencies may not be able to distinguish the error message from other help text present on the page.

EXPECTED RESULTS:

An error message should be clearly indicated as such, either through an error icon or the use of the word 'error' in front of the message.

STEPS TO REPRODUCE:

1. Enable screen reader.
2. Navigate to [URL](https://demo.civiform.dev/)
3. Navigate to and activate the ‘Apply’ for the Comprehensive sample program.
4. Navigate to and activate “What is your address?”.
5. Without filling any of the input fields, Navigate to and activate “Save and Next” .
6. Observe that the error message is only communicated using color and there is no presence of word 'error' or an error icon in front of the error message.

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93
- Experiments enabled: N/A
- Build name and version(optional): N/A

<img width="1343" alt="Screenshot 1" src="https://github.com/civiform/civiform/assets/86739416/0f4fb581-93f8-40b3-81de-fee9a8326fbd">
<img width="1399" alt="Screenshot2" src="https://github.com/civiform/civiform/assets/86739416/4d0af56c-3ca3-4e14-9a14-1bab6fa685f6">
<img width="1365" alt="Screenshot 3" src="https://github.com/civiform/civiform/assets/86739416/d164797e-fa44-4548-b7df-94a03e950696">
<img width="1407" alt="Screenshot 4" src="https://github.com/civiform/civiform/assets/86739416/7375a501-4dd6-4d6b-88e9-6a7d35d1a3e9">
<img width="1454" alt="Screenshot 5" src="https://github.com/civiform/civiform/assets/86739416/c0a4a993-5a44-463e-9ecc-5a74f1431b1b">
